### DISCUSSION TITLE
Facing issues with Allure Report after upgrading the Cypress version to latest one - 13.13

### DISCUSSION THREAD

**Discussion by skysamba** [allure-framework/allure2#2647] [Questions & Support] [answered]
Allure Report says, the report was generated successfully but doesn't seem to help with the numbers / data on the Report page. Any suggestions to overcome this?
![image](https://github.com/user-attachments/assets/63d717e0-c9a7-46f5-8742-ea774d5270ca)

**Comment by baev** [comment]
Please take a look at the contents of your `allure-results` folder. It should contain `UUID-result.json` files. Also, could you please specify the version of `allure-cypress` integration?

Additional information on how the Allure Report works can be found here: https://allurereport.org/docs/how-it-works/

**Reply by thilagavathysky** [reply]
It worked fine with Cypress 12.17.4 version. Recently, we updated the cypress version to 13.13.0. Post that, we are facing an issue with allure report generation. we are using an @shelex/cypress-allure-plugin: "2.34.0".

**Reply by baev** [reply, marked as answer]
`@shelex/cypress-allure-plugin` is a community-created integration, and as far as I know, it doesn't support the newer versions of Cypress. See https://github.com/Shelex/cypress-allure-plugin/issues/228

Please consider using the official integration instead.
